src/components/Card/PlaylistCard.js

```diff
@@ -75,18 +76,23 @@ PlaylistSummary.propTypes = {
   lessonsLeft: PropTypes.number.isRequired
 }
 
+const StyledPlaylistHeader = styled.div`
+  height: 290px;
+  .card-play-btn {
+    bottom: -2.5rem
+  }
+`
+
 export const PlaylistHeader = ({response, expanded}) => {
   const { lessons, progress, duration } = response
   const lessonsLeft = lessons.length - progress.completed_lessons.length
   const timeRemaining = secondsToString(getTimeLeft(duration, progress))
   return (
     <div>
-      <div className='relative w-100' style={{
-        height: '290px'
-      }}>
-        <PlayButton />
-        <Playlist playlist={buildPlaylistMeta(lessons, progress)} />
-      </div>
+      <StyledPlaylistHeader className='relative w-100'>
+        <PlayButton className='card-play-btn' />
+        <Playlist playlist={buildPlaylistMeta(lessons, progress)} card='playlist' />
+      </StyledPlaylistHeader>
       <PlaylistSummary timeRemaining={timeRemaining} lessonsLeft={lessonsLeft} />
```

Comment:
is this indication of progress not on the course cards? I'm almost positive it should be included along with the time remaining bit too

![](https://www.dropbox.com/home/_egghead-team/egghead-design/inprogress/browse_page/Design?preview=D-1-2-1-Browse-SIGNED-IN%402x.png.png)